elixir/apps/web/lib/web/auth.ex

```diff
@@ -285,7 +288,7 @@ defmodule Web.Auth do
   Fetches the session token from the session and assigns the subject to the connection.
   """
   def fetch_subject(%Plug.Conn{} = conn, _opts) do
-    params = take_sign_in_params(conn.query_params)
+    params = take_sign_in_params(conn.params)
```

Comment:
This is where the actual bug was.

Reply:
![](https://media.giphy.com/media/349qKnoIBHK1i/giphy.gif)